recipe/meta.yaml

```diff
@@ -112,12 +112,13 @@ outputs:
 
   - name: cuda-nvcc-impl
     build:
+      skip: true  # [win]
       # libnvvm.so gets corrupted by patchelf. No need to relocate as it is already relocatable
       binary_relocation: False
     run_exports:
       strong:
         - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
-    files:
+    files:                             # [linux]
```

Comment:
Same story as described in comment ( https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/5#discussion_r1297811780 )